GNUmakefile

```diff
@@ -36,10 +36,19 @@
 
 
 # --------------------------------------------------------------------------
-# Set libs3 version number, unless it is already set.
+# Set libs3 version number using git tags and optional developer PKG_PATCH
+# environment variable
+#
+
+# use phony .FORCE to ensure this is always regenerated
+.PHONY: .FORCE
```

Comment:
Isn't .PHONY: GIT-VERSION-FILE sufficient?  Do you really need a fake .FORCE dependency?